app/views/browse/config-map.html

```diff
@@ -58,7 +74,7 @@ <h1 class="contains-actions">
                     <truncate-long-text
                         content="value"
                         limit="1024"
-                        newlineLimit="20"
+                        newline-limit="20"
```

Comment:
This same error occurs at https://github.com/openshift/origin-web-console/blob/master/app/views/browse/config-map.html#L61

Not part of this PR but might be worth fixing here.

Reply:
Thanks, looks like that might be the same line as here?

Did a quick search for this, and I don't see any other references. I know @rhamilto fixed one in another place.